CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-[2.0.0-alpha.11]
+[2.0.0-alpha.14]
```

Comment:
Why did you change .11 to .14? 
And shouldn't the new info be in [unreleased] section above?